pkgs/development/python-modules/numpy-groupies/default.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  setuptools,
+  setuptools-scm,
+  numpy,
+  numba,
+  pandas,
+}:
+
+buildPythonPackage rec {
+  pname = "numpy-groupies";
+  version = "0.11.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ml31415";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Eu+5SR28jIasKe1p7rvbq2yo3PGZRQWWdG3A5vGhnyM=";
+  };
+
+  build-system = [
+    setuptools
+    setuptools-scm
+  ];
+
+  dependencies = [ numpy ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    numba
+    pandas
+  ];
```

```suggestion
  ];
  
  pythonImportsCheck = [ "numpy_groupies" ];
```
Could you add `pythonImportsCheck` for a smoke test?

> pythonImportsCheck is set. This is still a good smoke test even if pytestCheckHook is set.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines